snd_devices.py

```diff
@@ -0,0 +1,23 @@
+# from bluesky import RunEngine
+
+# from pcdsdevices.daq import make_daq_run_engine
+
+from hxrsnd.sndmotor import SamMotor
+from hxrsnd.sequencer import SeqBase
+from hxrsnd.sndsystem import SplitAndDelay
+
+# Base PV
+pv_base = "XCS:SND"
+
+# Instantiate the whole system
+snd = SplitAndDelay(pv_base)
+daq = snd.daq
+
+# Create a RunEngine
+# RE = RunEngine({})
+# RE_daq = make_daq_run_engine(snd.daq)
```

Comment:
Is this supposed to be commented out?

Reply:
From the PR description:
- comment out references to `bluesky`
- references to `RE` are currently confusing for the user, until we implement this fully

Until we've shown everyone how to do scans with RE, it shouldn't be visible in the user space. I did this as a result of a conversation I had with Rajan where he seemed very confused about why there were `RE` instances and references to `bluesky` when they weren't being used in production.

As an aside, these lines make it so we have a total of three `RunEngine` instances in the user space (including `snd.RE`), which doesn't seem kosher. Maybe there's a better way to implement `RE_daq` to avoid this.

Reply:
If there's some disagreement on the best thing to do with the `RE` instances, rest easy in that the only parts of this PR sitting in the prod area are the additions to the `scripts.py` file

Reply:
I'm slowly reacquainting myself with everything we did last year. Yeah, I think this makes sense.